test/module/irohad/torii/torii_service_test.cpp

```diff
@@ -247,12 +252,13 @@ TEST_F(ToriiServiceTest, StatusWhenBlocking) {
   commit_notifier_.get_subscriber().on_next(commit);
   block_notifier_.get_subscriber().on_next(block);
 
+  auto client3 = client2;
```

Comment:
May be it would be better to move copying/assigning cases into a separate test?
Also could you please add a test about possibility of same object reusing? Before this pr it resulted with a segfault.

Reply:
Check this one
https://github.com/hyperledger/iroha/blob/8297222c72db0bc5e487e4de474b2e5fa01633cd/test/module/irohad/torii/torii_service_test.cpp#L175